Makefile

```diff
@@ -52,5 +52,6 @@ clean:
 .PHONY: dev
 dev:
 	go get github.com/pilu/fresh
-	docker-compose up
+	docker-compose start
```

Comment:
@tsmaeder The issue you had with docker-compose was that it 'hang'. Up hangs by default, start does not. But they are semantically different as well. 'up -d' sets it in daemon mode so it won't hang. --force-recreate just recreates the container even if the configuration hasn't changed so you start from a clean 'build'.